custom/conf/app.ini.sample

```diff
@@ -486,6 +486,7 @@ MODE = console
 BUFFER_LEN = 10000
 ; Either "Trace", "Debug", "Info", "Warn", "Error", "Critical", default is "Trace"
 LEVEL = Trace
+REDIRECT_MACARON_LOG = false
```

Comment:
Should we let it default as true even it will break the compatible.

Reply:
We probably need to go through our integration tests and make them log a bit more, and add the `-v` command line option - otherwise they're basically silent to drone except for errors with this option turned on.

That of course may be a good thing...

I think this is an owners decision. Another option is to just leave it for this PR and put out a new PR to discuss turning it on.